app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,45 @@
+module SessionsHelper
+  # Logs in the given user
```

Comment:
Where dis you copy this code?)
Are you able to rewrite it by yourself?))

Reply:
There are parts of code I used on my previous project. It works fine anywhere